README.md

```diff
@@ -2,64 +2,69 @@
 
 > ESLint [shareable config](http://eslint.org/docs/developer-guide/shareable-configs.html) developed and maintained by [nimble](https://nimblehq.co/)
 
-## Installation
+## Usage
 
-```
-$ npm install --save-dev @nimblehq/eslint-config-nimble
-```
+The configurations are separated into dedicated packages:
+- [eslint-config-nimble](/packages/eslint-config-nimble): ESLint base rules
```

Comment:
```suggestion
The configurations are separated into dedicated packages:

- [eslint-config-nimble](/packages/eslint-config-nimble): ESLint base rules
```

Reply:
Fix this and other comments in https://github.com/nimblehq/eslint-config-nimble/pull/20/commits/d74720d74295f92913a3eb8f5efc67fcb5b07783